test/framework/cluster.go

```diff
@@ -1263,7 +1263,7 @@ func (e *ClusterE2ETest) VerifyAdotPackageDaemonSetUpdated(packageName string, t
 		e.T.Fatalf("waiting for adot package update timed out: %s", err)
 	}
 
-	time.Sleep(15 * time.Second) // Add sleep to allow daemonset to be created
+	time.Sleep(30 * time.Second) // Add sleep to allow daemonset to be created
```

Comment:
Something similar to this maybe?
```
    e.T.Log("Waiting for control plane to be ready")
    err := retrier.New(5 * time.Minute).Retry(func() error {
        return e.KubectlClient.ValidateControlPlaneNodes(context.Background(), e.cluster(), e.ClusterName)
    })
    if err != nil {
        e.T.Fatal(err)
    }
```

Reply:
Thanks. I've made an update.

I tested this locally and noticed that w/o the sleep the downstream checking log step can have issues too, so added a retrier there too.